### DISCUSSION TITLE
Using PyFunceble ? Let us know!

### DISCUSSION THREAD

**Discussion by funilrys** [funilrys/PyFunceble#169] [Show and tell]
Hello,

I want to collect a list of people, organizations, or entities using PyFunceble. 

Drop a comment here with your links and I will put them on our landing page at https://pyfunceble.github.io/#/they-use-d-it and in the documentation. 

Stay safe and healthy!

**Comment by spirillen** [comment]
From https://github.com/spirillen/PyFunceble/pull/25#discussion_r553555644

There will actually be 2

.. _porn-records : https://www.mypdns.org/project/view/10/ (https://github.com/mypdns/porn-records  | https://www.mypdns.org/source/PornRecords/ )

Not sure which to set primary from [mypdns.org](https://www.mypdns.org/).... but leaning to the project url

**Reply by spirillen** [reply]
I decided to link to the umbrella one: <https://www.mypdns.org/project/view/10/>
